test/removeFromTeam.js

```diff
@@ -47,6 +79,56 @@ describe("removing a user from a team", function() {
         );
     }));
 
-    // TODO: add tests for more hidden groups other than just in announcements
+    it("should remove the user from event hidden groups", coroutine(function*() {
+        let date = new Date();
+        let events = yield sessions[0]("GET",
+            `/events/startYear/${date.getFullYear() - 1}/startMonth/0
+                /endYear/${date.getFullYear() + 1}/endMonth/0`);
+        assert.equal(events[0].audience.users.length, 1,
+            "a user was removed from event hidden group"
+        );
+        assert.equal(events[0].audience.users[0], data.users[0]._id,
+            "the correct user was removed from event hidden group"
+        );
```

Comment:
These 2 asserts can be combined with `deepEqual`. For example

```assert.deepEqual(events[0].audience.users, [data.users[0]._id], "message");```

The same goes for some of the pairts of `assert`s below.